src/lib/directive.js

```diff
@@ -85,7 +85,7 @@ const _ = require('lodash');
         return target.email;
       }).join(',');
 
-      return new URL('/mailto/?uri=mailto:' + encodeURIComponent(recipients), $window.location.origin).toString();
+      return new URL('/mailto/?uri=mailto:' + encodeURIComponent(recipients), window.openpaas.MAILTO_SPA_URL).toString();
```

Comment:
```suggestion
      return new URL('/mailto/?uri=mailto:' + encodeURIComponent(recipients), window.openpaas && window.openpaas.MAILTO_SPA_URL || $window.location.origin).toString();
```

this should continue to work if we don't have an openpaas.js env file ( MAILTO_SPA_URL will be undefined ) or using a same domain for all SPAs